requirements/serve.txt

```diff
@@ -1,6 +1,2 @@
-fastapi
 gradio<4.0.0
```

Comment:
could we add protobuf as required here 
https://github.com/InternLM/lmdeploy/blob/cc316a3e582e4a2d66d6e72f5fb41bb36a2f24ea/lmdeploy/serve/turbomind/chatbot.py#L12

Reply:
Sure. Is there any constraints on its version?

Reply:
No version restriction.

Reply:
OK. `protobuf` is added to `serve.txt`